test/sanity/issue6522-new-win-policy/test.py

```diff
@@ -0,0 +1,42 @@
+import time
+import os
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys 
+from selenium.webdriver.common.alert import Alert
+
+chrome_options = Options()
+testdir = os.path.dirname(os.path.abspath(__file__))
+chrome_options.add_argument("nwapp=" + testdir)
+
+def click(element_id, num):
+    element_id = driver.find_element_by_id(element_id)
+    time.sleep(1)
```

Comment:
Please find ways to remove time.sleep

Reply:
Removed it.

Reply:
does it imply a race condition? it could lead to unstable results in slow/fast systems.